doozer/doozerlib/olm/bundle.py

```diff
@@ -29,77 +29,57 @@ class OLMBundle(object):
+                 dry_run: Optional[bool] = False,
+                 brew_session: Optional[ClientSession] = None):
         self.runtime = runtime
         self.dry_run = dry_run
         self.brew_session = brew_session or runtime.build_retrying_koji_client()
-        self.operator_nvr: Optional[str] = None
         self.operator_dict: Optional[dict] = None
-        self.operator_repo_name: Optional[str] = None
-        self.operator_build_commit: Optional[str] = None
 
-    def find_bundle_for(self, operator_build: Union[str, dict]) -> str:
-        """Check if a bundle already exists for a given `operator_build`.
-
-        :param operator_build: Operator build dict or NVR (format: my-operator-container-v4.2.30-202004200449)
-        :return: NVR of latest found bundle build, or None if there is no bundle build nvr
-                        corresponding to the last operator build's nvr.
-        """
-        if isinstance(operator_build, str):
-            self.operator_nvr = operator_build
+        if isinstance(operator_nvr_or_dict, dict):
+            self.operator_dict = operator_nvr_or_dict
         else:
-            self.operator_nvr = operator_build["nvr"]
-            self.operator_dict = operator_build
+            build = brew.get_build_objects([operator_nvr_or_dict], self.brew_session)[0]
+            if not build:
+                raise IOError(f"Build {self.operator_nvr} doesn't exist in Brew.")
+            self.operator_dict = build
+
+    @property
+    def operator_nvr(self):
+        return self.operator_dict['nvr']
+
+    @property
+    def operator_repo_name(self):
+        source_url = urlparse(self.operator_dict['source'])
+        # /git/containers/ose-cluster-kube-descheduler-operator => containers/ose-cluster-kube-descheduler-operator
+        return f"containers/{source_url.path.rstrip('/').rsplit('/')[-1]}"
+
+    @property
+    def operator_build_commit(self):
+        source_url = urlparse(self.operator_dict['source'])
+        return source_url.fragment
 
-        self.get_operator_buildinfo()
+    def find_bundle_image(self) -> str:
+        """Check if a bundle already exists for the nvr that was used to init the OLMBundle object
 
+        :return: NVR of latest found bundle build, or None if there is no bundle build nvr
+                        corresponding to the nvr used to init the OLMBundle object
+        """
         prefix = f"{self.bundle_brew_component}-{self.operator_dict['version']}.{self.operator_dict['release']}-"
         bundle_package_id = self.brew_session.getPackageID(self.bundle_brew_component, strict=True)
         builds = self.brew_session.listBuilds(packageID=bundle_package_id, pattern=prefix + "*", state=brew.BuildStates.COMPLETE.value,
                                               queryOpts={'limit': 1, 'order': '-creation_event_id'}, completeBefore=None)
         return builds[0]['nvr'] if builds else None
 
-    def get_latest_bundle_build(self):
```

Comment:
unused, old way of doing things. We didn't delete this when we moved to ^ new way of doing things